hazelcast/src/test/java/com/hazelcast/map/EntryProcessorTest.java

```diff
@@ -1444,6 +1444,7 @@ public void testReadOnlyEntryProcessorDoesNotCreateBackup() {
     private void testEntryProcessorWithPredicate_updatesLastAccessTime(boolean accessExpected) {
         Config config = withoutNetworkJoin(smallInstanceConfig());
         config.getMapConfig(MAP_NAME)
+                .setPerEntryStatsEnabled(true)
```

Comment:
I'm thinking of parameterizing these tests like in ExpirationTimeTest so that we make the tests that can only work on when perEntryStats is enabled more visible. Of course, this will extend the running time of the tests. So, it is totally ok for me to keep these tests as they are. I'm just writing to consider.